package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^1.4.0",
+    "moment": "^2.29.4",
```

Comment:
Really nice that you've started learning about `axios` 👍  
Not a good practice to push dependencies, can you do a bit of study on how to manage dependencies and exclude them from version control using Git.